redis_enterprise/metadata.csv

```diff
@@ -0,0 +1,180 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,
```

Comment:
We've improved validations, so any future commit will catch this automatically. 
But it looks like you have an extra `,` in this file, causing it to have 11 rows instead of the expected and required 10. Could you please fix this?

Reply:
done